code/ui/manager/src/components/sidebar/Search.tsx

```diff
@@ -126,6 +126,7 @@ const FocusKey = styled.code(({ theme }) => ({
   borderRadius: 3,
   userSelect: 'none',
   pointerEvents: 'none',
+  fontFamily: 'inherit',
```

Comment:
This is super interesting! I think you spotted a problem that has been there for quite some time cc @MichaelArestad 

The font-family for `code` elements is ui-monospace, while the rest of Storybook uses Nunito sans. I think it'd be better to handle this elsewhere, so we can also fix the issues in the setting tooltip. For instance, it looks like this now:
![image](https://github.com/storybookjs/storybook/assets/1671563/a9c590fa-a60c-4d26-8041-660a4429e7b4)

with the font fixed, it will look like this:
![image](https://github.com/storybookjs/storybook/assets/1671563/0df324c3-6824-4404-b1d2-c4838e84de9f)